src/components/Recommend/Recommend.style.tsx

```diff
@@ -129,7 +130,7 @@ export const FooterHalf = styled.div`
 `;
 
 //TODO (chelsi)-make button and not div
-export const FooterLink = styled.div`
+export const FooterLink = styled(LinkButton)`
```

Comment:
nit: this component's hover state (gray background) is out of sync with that of the footer link in compare (the 'view all' text). maybe nix the hover state or apply this component to compare's footer?